pkg/router/router.go

```diff
@@ -59,6 +60,7 @@ const ExtensionDisplayNameKey = "tekton-dashboard-display-name"
 const ExtensionRoot = "/v1/extensions"
 
 var webResourcesDir = os.Getenv("WEB_RESOURCES_DIR")
+var webResourcesPattern = regexp.MustCompile("^/([[:alnum:]]+\\.)?[[:alnum:]]+\\.(js)|(css)|(png)$")
```

Comment:
The favicon (the only png I see loaded) does not include a hash, so if we can exclude it for now this PR should be good to go. We'll need to make some changes on the frontend for the favicon so I'll create an issue to track that separately.

```suggestion
var webResourcesPattern = regexp.MustCompile("^/([[:alnum:]]+\\.)?[[:alnum:]]+\\.(js)|(css)$")
```

Reply:
We also load the Tekton logo png (in the upper right corner).  I'll add a specific exclusion for the favicon.